### ISSUE TITLE
Predict point by point is very slow

### ISSUE THREAD

**Issue by sporubcan** [keras-team/keras#11006]
Hi,
I am using Keras LSTM on time series prediction (let time series is y[n]). Network input is time shifted vector y[0], y[-1], y[-2] etc.... After network training, I calculate one step prediction, then add prediction to input vector on first place and release last value. So i get prediction to e.g. 50 future points. Problem is, that predict method is very slow for this case. I noticed, that prediction time does change only a little with changing network size (for small networks - e.g.  1-50 LSTM cells). I know, that prediction of whole array of input of vectors is significantly faster, but that is not what i want. I tried to change Keras backend to CNTK or Theano, but it was worse than Tensorflow. 
Is there possibility to speed it up? What about of adding fast predict method for only one input vector? Would be there a way to use Tensorflow or CNTK directly without running through Keras?

Model I use (example):
`model = Sequential()`
`model.add(LSTM(units=10, input_shape = (49,1), return_sequences = False)) `
`model.add(Dropout(0.2))`
`model.add(Dense(1))`
`model.add(Activation("linear"))`
`model.compile(loss="mse", optimizer="adam")`
`model.summary()`

My function for multiple step prediction (I know, converting array to lists an backwards is not optimal):

`def createPrediction(testData, n_future_steps):`
`_x = array([testData])`
`_result = []`
`_for i in range(0, n_future_steps):`
` __prediction = model.predict(x)`
` __result.append(prediction[0, 0])`
` __p = [prediction[0, 0]] + x[0, :-1, 0].tolist()`
` __r = array(p)`
` __x = r.reshape(1, -1, 1)`
`_result = array(result)`
`_return result`
`
Keras version is 2.2.2
Tensorflow version is 1.5.0 (I have problems with newer versions)

Profiling shows, that bottle neck ist in _pywrap_tensorflow_internal.TF_Run method.
